packages/execution-environments/src/common/endowments/network.ts

```diff
@@ -25,7 +25,10 @@ const createNetwork = () => {
     (callback) => callback(),
   );
 
-  const _fetch: typeof fetch = async (
+  // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+  const fetch = require('isomorphic-unfetch');
```

Comment:
Doesn't `isomorphic-unfetch` use `XMLHttpRequest` behind the scenes? We should probably try to use `fetch` where possible?